liberty-maven-plugin/src/it/tests/basic-it/pom.xml

```diff
@@ -174,6 +174,10 @@
                         <type>zip</type>
                     </assemblyArtifact>
                     <serverName>test</serverName>
+                    <!--Testing if passing an empty value fails-->
```

Comment:
Change the comment to say:
<!--Testing if passing an empty value succeeds -->
or 
<!--Testing if passing an empty value does not fail-->